src/shared/components/Header/HomeHeader/HomeHeader.css.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+export const HeaderWrapper = style({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  padding: '1.1rem 2.4rem',
+  width: '375px',
+  boxSizing: 'border-box',
+  backgroundColor: ' #FFF',
+  flexShrink: 0,
+  position: 'fixed',
+  top: 0,
+  zIndex: 1,
+});
+export const HeaderTap = style({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  gap: '2.5rem',
+  alignItems: 'center',
+});
+export const HeaderTxt = recipe({
+  base: {
+    color: '#0E0E0E',
+    fontSize: '2rem',
+    fontWeight: 400,
+    lineHeight: '2.3rem',
+    letterSpacing: '-0.04rem',
```

Comment:
```suggestion

import { fontStyle } from '@/shared/styles/fontStyle';

...fontStyle('h6_re_20')
```
폰트도 토큰화 시켜두어서 위처럼 fontStyle import 하시고 피그마에 있는 폰트명으로 변경 해주시믄 감사하겠습니다! 전부 안 써두 됩니닷 . ㅎ

Reply:
수정했습니당! (❁´◡`❁)